core/src/apps/kaspa/addresses.py

```
@@ -39,17 +39,20 @@ def is_valid(cls, prefix: str) -> bool:
 
 def encode_address(
     node: bip32.HDNode,
-    schema: str,
+    scheme: str,
     prefix: str = Prefix.Mainnet,
+    use_tweak: bool = True,
```

Ruff (0.11.9)

Boolean-typed positional argument in function definition

(FBT001)